Backend-llm/tests/test_streaming_manual.py

```
@@ -0,0 +1,244 @@
+                        if data.get("done"):
+                            end_s = datetime.now()
+                            break
+    
+    time_s_total = (end_s - start_s).total_seconds() * 1000
+    time_s_ttft = (first_token_time - start_s).total_seconds() * 1000 if first_token_time else 0
+    
+    print(f"   ✓ Time to first token: {time_s_ttft:.2f}ms")
+    print(f"   ✓ Total time: {time_s_total:.2f}ms")
+    
+    # Comparison
+    print(f"\n📊 Comparison:")
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)